css/ampshared.css

```diff
@@ -155,6 +155,16 @@ i-amphtml-sizer {
   display: block !important;
 }
 
+/*
+ * Disable `i-amphtml-sizer` when aspect ratio is supported by both the
+ * borwser and the optimizer.
+ */
+@supports (aspect-ratio: 1 / 1) {
```

Comment:
Have we validated this is completely ignored in unsupported browsers? Am not familiar with how older engines treat this at-rule.

Reply:
Yes. Works correctly everywhere, including IE.